modules/nf-core/bedtools/intersect/meta.yml

```diff
@@ -26,6 +26,10 @@ input:
   - extension:
       type: value
       description: Extension of the output file. It is set by the user and corresponds to the file format which depends on arguments (e. g., ".bed", ".bam", ".txt", etc.).
+  - chrom_sizes:
```

Comment:
Add `meta` to chrom_sizes here and remove `extension`, but leave a note about the `ext.suffix` use